capi/include/solana.h

```diff
@@ -0,0 +1,163 @@
+// Copyright 2018 Solana Labs, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+// Warning, this file is autogenerated by cbindgen. Don't modify this manually.
```

Comment:
If this file is auto generated, can we avoid committing it?

Reply:
My thinking was that it would make it easier for C developers to understand the API on github with this comitted, but I can also remove it and add a README explaining how to build the static lib and generate the header file (which I'm realizing should happen anyway).